### ISSUE TITLE
Phase Two: Automate status changes (pending scope)

### ISSUE THREAD

**Issue by playcock** [stellarbb/devops#40]
### Feature Request

Ryan, Matt, Russ think and discuss what the following means for Automate status changes:
   o Read
   o In Progress
   o On Hold
   o Waiting on Feedback
      - Provide checkbox when sending public message to mark as Waiting on Feedback
